standards/peer-review.md

```diff
@@ -1,41 +1,38 @@
-## Pull Request Workflow
+## PR Review Guidelines & Best Practices
 
-- All Pull Requests to NYPL repositories MUST be reviewed prior to merging (caveat below).
-  - PRs MUST NOT be merged if a reviewer requests changes
-  - Teams MAY agree to specific written policies as to conditions when reviews are not required. Such a policy MUST be approved by an Engineering Manager or Director.
-  
-- Engineers MUST mark at least one (1) developer as a reviewer and SHOULD request reviews from all relevant engineers
-  - Reviewers MAY come from any team within NYPL Digital
-  - Teams MAY create [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) files to establish a default group of reviewers
-  - Experience SHOULD NOT be taken into account when requesting review
-  - Requests for specific feedback MAY be requested by tagging specific engineers in PR comments
+These guidelines establish the principles and processes for Pull Request (PR) reviews. Following them ensures we maintain high code quality, share knowledge effectively, and foster a collaborative and supportive engineering culture.
 
-- Engineers marked as reviewers SHOULD complete the review in a timely manner
-  - This period SHOULD be agreed upon within individual development teams, 24-48 hours has proven to be a reasonable starting place
-  - Code review SHOULD take some time (more than a quick LGTM) but should not take more than 1-2 hours
-  - If a code review is very lengthy or complex, reviewers MAY request that the PR be broken up into smaller pieces and/or request a walkthrough/live review session
- 
-- If changes are requested (or made in response to questions) the requesting engineer SHOULD re-request review when ready
-  - This helps reviewers track the current status of open PRs and plan time for review work
-  - Additional rounds of review SHOULD be completed until the PR can be either merged or closed
-  - To help track PR status teams MAY use GitHub labels (e.g. `Changes Requested`, `Needs Review` and `Ship It`) to help socialize status.
+All pull requests (PRs) **must** be reviewed by at least one other engineer before being merged. Exemptions to this rule require a specific, written policy agreed upon by the team and approved by an Engineering Manager or Director.
```

Comment:
Nit: I think we should follow the readme's key word instruction and write "must" in all-caps.